lib/module.js

```diff
@@ -460,29 +461,23 @@ Module.runMain = function() {
 };
 
 Module._initPaths = function() {
-  const isWindows = process.platform === 'win32';
+  modulePaths = [];
```

Comment:
@meinklaus `_initPaths` is called only once and modulePaths is `undefined` at the beginning.